src/Utility/Text.php

```diff
@@ -1034,7 +1034,7 @@ public static function parseFileSize($size, $default = false)
             $i = array_search(substr($size, -1), ['K', 'M', 'G', 'T', 'P']);
         }
         if ($i !== false) {
-            $size = substr($size, 0, $l);
+            $size = (float)substr($size, 0, $l);
```

Comment:
there was not error about this

Reply:
substr() returns int, so this makes sense.

Reply:
Actually, it can also return bool false, what in that case? Would just be 0 then to go into the operation below, which seems fine as well.